### PR TITLE
Sourabh aws observability

### PR DESCRIPTION
**Pull request by SumoSourabh**
- added check for errors
- added monitor export support
- added support to install JSON from custom s3 url

### CODE REVIEW

File: sumologic-app-utils/src/sumoresource.py [hunk 1]
```diff
@@ -631,27 +650,22 @@ def create_by_install_api(self, appid, appname, source_params, folder_name, *arg
             return {"APP_FOLDER_NAME": content["name"]}, app_folder_id
         else:
             print("%s installation failed." % appname)
-            response.raise_for_status()
+            raise Exception(response.text)
```
Comment (himanshu219) [inline sumologic-app-utils/src/sumoresource.py:653]: response for status will raise specific error like Connection, Timeout etc any particular reason for creating new Exception?

File: sumologic-app-utils/src/sumoresource.py [hunk 2]
```diff
@@ -1249,6 +1267,85 @@ def extract_params(self, event):
...
+
+    def _get_root_folder_id(self):
+        response = self.sumologic_cli.get_root_folder()
+        return response["id"]
+
+    def import_monitor(self, folder_name, monitors3url, variables, suffix_date_time):
+        date_format = "%d-%b-%Y %H:%M:%S"
+        root_folder_id = self._get_root_folder_id()
+        content = self._get_content_from_s3(monitors3url, variables)
+        content["name"] = folder_name + " " + datetime.utcnow().strftime(date_format) if suffix_date_time \
```
Comment (himanshu219) [inline sumologic-app-utils/src/sumoresource.py:1302]: is it expected? concatenating folder name with folder name if no suffix_date_time

**Review by himanshu219 — APPROVED** [review]
LGTM

File: sumologic-app-utils/src/sumoresource.py (hunk 2 above)
Comment (SumoSourabh) [reply sumologic-app-utils/src/sumoresource.py:1302]: total condition is in else

File: sumologic-app-utils/src/sumoresource.py (hunk 1 above)
Comment (SumoSourabh) [reply sumologic-app-utils/src/sumoresource.py:653]: Yes, did not exactly remember but the error was not looking correct so had to do it

File: sumologic-app-utils/src/sumoresource.py (hunk 2 above)
Comment (himanshu219) [reply sumologic-app-utils/src/sumoresource.py:1302]: got it, I guess "if" has higher precedence over +.

**Pull request merged**